mongodb/types/automation_agent_config.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"path"
+	"reflect"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+// AutomationAgentConfig holder for Automation Agent Config
+type AutomationAgentConfig struct {
+	Binary    string                 `json:"binary,omitempty"`
+	BaseURL   string                 `json:"baseurl,omitempty" automation:"mmsBaseUrl"`
```

Comment:
mms_base_url

Reply:
in general, (I think) it will be easier for users to reason what these translate to if the names in the TF config are as similar as possible to the ones in the agent's `local.config`, OM's `conf-mms.properties`, etc.

Reply:
see other comment about naming